tools/iceworks-app/package.json

```diff
@@ -24,6 +24,7 @@
     "electron-updater": "^4.1.2",
     "execa": "^1.0.0",
     "ice-npm-utils": "^1.1.0",
+    "npm": "^6.9.0",
```

Comment:
A better way is to have a fixed version number.

Reply:
OK